### PR TITLE
design-role-view

### PR DESCRIPTION
**Pull request by kienpham9819**
## Related Tickets:
- [#29914](https://edu-redmine.sun-asterisk.vn/issues/29914)

## WHAT this PR does?
- design role-permission view

## Notes Impacted Areas:
- N/A.

## Attachments:
- Screenshot from 2020-10-05 10-35-06
- Screenshot from 2020-10-05 10-35-22

### CODE REVIEW

File: app/Http/Controllers/RoleController.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+
+    public function edit(Role $role)
+    {
+        $permissions = Permission::all();
+
+        return view('users.admin.role_edit', compact(['role', 'permissions']));
```
Comment (TDM1911) [inline app/Http/Controllers/RoleController.php:40]:
```suggestion
        return view('users.admin.role_edit', compact([
            'role', 
            'permissions',
        ]));
```

File: app/Http/Controllers/RoleController.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(RoleRequest $request, Role $role)
+    {
+        $role->update([
+            'name' => $request->name,
+            'slug' => Str::slug($request->name, '-'),
```
Comment (TDM1911) [inline app/Http/Controllers/RoleController.php:54]:
```suggestion
            'slug' => Str::slug($request->name),
```
khong can dung "-".

**Comment by kienpham9819** [comment]
ready

File: resources/views/users/admin/role_edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th>#</th>
+                            <th>{{ trans('role.name') }}</th>
+                            <th>{{ trans('role.option') }}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach($permissions as $key => $permission)
```
Comment (huyenttm-0901) [inline resources/views/users/admin/role_edit.blade.php]: convention

File: resources/views/users/admin/role_edit.blade.php [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                                <td>
+                                    <label for="{{ $permission->slug }}">
+                                        {{ $permission->name }}
+                                    </label>
+                                </td>
+                                <td>
+                                    <input type="checkbox"
+                                        id="{{ $permission->slug }}"
+                                        name="permission[]"
+                                        @if($role->permissions->contains($permission))
```
Comment (huyenttm-0901) [inline resources/views/users/admin/role_edit.blade.php]: tương tự

**Comment by kienpham9819** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**